src/test/kotlin/com/ampnet/userservice/grpc/GrpcUserServerTest.kt

```diff
@@ -167,6 +172,58 @@ class GrpcUserServerTest : TestBase() {
         }
     }
 
+    @Test
+    fun mustReturnUsersWithExtendedInfo() {
+        suppose("platform manager exists") {
+            testContext.user = createUser(UUID.randomUUID(), role = UserRole.PLATFORM_MANAGER)
+            Mockito.`when`(userRepository.findByCoopAndUuid(COOP, testContext.user.uuid))
+                .thenReturn(Optional.of(testContext.user))
+        }
```

Comment:
I think there is no more need for a `Platform manager`.

Reply:
It's not unnecessary, but it may be redundant. I'll remove it.